app/assets/javascripts/snowcrash/modules/issues.js.coffee

```diff
@@ -1,8 +1,9 @@
 //= require snowcrash/modules/issues/evidence
 //= require snowcrash/modules/issues/importer
+//= require snowcrash/modules/issues/merge
 //= require snowcrash/modules/issues/table
 //= require snowcrash/modules/issues/tag-input
-//= require snowcrash/modules/issues/merge
+//= require snowcrash/modules/issues/summary
```

Comment:
For the sake of consistency I'd make this a "page" and not a "module" and I'd put it under "pages/projects" as this is where the code is used (and not in IssuesController).